### PR TITLE
Add deployment to npm through circle ci

### PR DESCRIPTION
**Pull request by msecret**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+machine:
+  node:
+    version: 5.3.0
```
Comment (ntwb) [inline circle.yml:3]: Why NodeJS 5.3.0? It's neither the 4.x.x LTS or 6.x.x Current release? https://nodejs.org/en/

Comment (el-mapache) [inline circle.yml:3]: It was the last version I used in a major app, in March of this year. I see that 6.0 was released in April, so we can switch to 6.2.0 if necessary. There wasn't a real reason beyond that. Are we supposed to use 4.x.x in our projects?

Comment (ntwb) [inline circle.yml:3]: Cool 👍 , If Circle CI use NVM for NodeJS version management then changing this to `4` or `6` will get you the latest version from each respective branch (and save you from having to update it too often) once you've decided which branch you'll use for the project 😄

**Comment by msecret** [comment]
I like the idea of switching to `6.x.x` or `6.0.x` if we want to be more conservative.

File: circle.yml (hunk 1 above)
Comment (msecret) [inline circle.yml:3]: I like the idea of switching to 6.x.x or 6.0.x if we want to be more conservative.

Comment (ntwb) [inline circle.yml:3]: Yep, [Circle CI uses NVM](https://github.com/circleci/circleci-docs/blob/master/jekyll/_docs/configuration.md#nodejs-version), change your `5.3.0` to `6` and you'll always have the latest version of the 6.x.x branch, stylelint also tests against both the [4.x LTS and 6.x Current](https://github.com/stylelint/stylelint/blob/master/.travis.yml#L5) branches FWIW:

Comment (el-mapache) [inline circle.yml:3]: Seems like a good idea to bring the project in line with the newer version, thanks for the heads up @ntwb! 👍

**Pull request merged**